.github/workflows/master.yml

```diff
@@ -34,15 +33,30 @@ jobs:
         run: yarn install
       - name: Build
         run: yarn build
+  lhci:
+    name: Lighthouse check
+    if: ${{ github.event.label.name == 'lighthouse-check' }}
```

Comment:
With this `if` the check was skipped even with the `lighthouse-chec` label

Reply:
This was skipped because here I am checking for a single label (if there are more than one selected then it returns false)